mlfinlab/online_portfolio_selection/universal_portfolio.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, number_of_experts, weighted='hist_performance', k=1):
         :param number_of_experts: (int) Number of total experts
         :param weighted: (str) Capital allocation method. 'hist_performance': Historical Performance,
                                'uniform': Uniform Weights, 'top-k': Top-K experts.
+        :param k: (int) Number of experts to choose your portfolio. Only necessary if weighted is
```

Comment:
Is the `k` the best name? Maybe having a more intuitive name will boost user experience.